DependencyInjection/CydricknOpenApiValidatorExtension.php

```diff
@@ -58,8 +63,17 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
         ]);
         $container->setDefinition('cydrickn.openapi_validator.validator', $validator);
 
+        if ($conditionInfo = $mergedConfig['condition'] ?? null) {
+            $container->setDefinition('cydrickn.openapi_validator.condition', (function() use ($conditionInfo) {
+                $type = array_keys($conditionInfo)[0];
+                $args = array_values($conditionInfo)[0];
+                return new Definition(self::CONDITIONS[$type], [$args]);
+            })());
+        }
+
         $validatorListener = new Definition(ValidatorListener::class, [
             new Reference('cydrickn.openapi_validator.validator'),
+            !empty($mergedConfig['condition']) ? new Reference('cydrickn.openapi_validator.condition') : null
```

Comment:
Hi @akaffenberger ,

Maybe you can change this to.

```suggestion
           $conditionInfo !== null ? new Reference('cydrickn.openapi_validator.condition') : null
```